scm-ui/ui-webapp/src/users/components/table/Details.tsx

```diff
@@ -62,8 +62,10 @@ class Details extends React.Component<Props> {
             </td>
           </tr>
           <tr>
-            <th>{t("user.type")}</th>
-            <td>{user.type}</td>
+            <th>{t("user.externalFlag")}</th>
+            <td>
+              <Checkbox checked={!!user.external} />
```

Comment:
Is this not just `user.external`?

Reply:
No, because external is optional

Reply:
How can this be optional? The User object has `boolean` field which is `false` or `true`, but never `null` or `undefined`.